experiment_dirs

```diff
@@ -60,4 +60,9 @@ for group in $*; do
     esac
 done
 
+EXCLUSIONS=(/g/data/ik11/outputs/access-om2-01/01deg_jra55v140_iaf_cycle4_MWpert)
+for exclude in ${EXCLUSIONS[@]}; do
+    EXPT_DIRS=("${EXPT_DIRS[@]/$exclude}")
```

Comment:
I'm not familiar with this syntax, but does it need another / after $exclude?

Reply:
No, to my knowledge it does not. Maybe it would be clearer to use `%` instead of `\`, as in this case it produces the same result.

Obviously here I'm assuming that `$exclude` contains a full path to some folder, not the parent of several folders that are to be excluded...